src/weather.cpp

```diff
@@ -68,6 +69,32 @@ void weather_effect::glare()
         }
     }
 }
+/**
+ * Snow Glare.
+ * Causes glare effect to player's eyes if they are not wearing applicable eye protection.
+ */
+void weather_effect::snow_glare()
```

Comment:
Is this a copy of `weather_effect::glare()`? Maybe you should move shared code to some function and use it in both methods?

Reply:
No, it doesn't work, I'll keep trying.

Reply:
ok got it now